schedule/yast/btrfs/btrfs_sle_libstorage-ng.yaml

```diff
@@ -38,12 +38,25 @@ schedule:
   - installation/first_boot
   - console/system_prepare
   - console/force_scheduled_tasks
-  - console/check_resume
+  - '{{hibernation_enabled}}'
```

Comment:
you just need 1 placeholder to replace as both test would not run at the same time.

Reply:
It runs both on x86_64 and other architectures, so it needs to be this way. Or I am I missing something ?

Reply:
we can just do it in one block, like in other examples in the repo, it is designed in that way:
```
  validate_hibernation:
    ARCH:
      aarch64:
        - console/hibernation_disabled
      ppc64le:
        - console/hibernation_disabled
      s390x:
        - console/hibernation_disabled
      x86_64:
        - console/hibernation_enabled
 ```

Reply:
Ah, I did not know, thanks !